database/migration/migrateData.ts

```diff
@@ -916,149 +916,151 @@ const migrateData = async () => {
+              r.review_assignment_id,
+              r.stage_number,
+              r.level_number,
+              r.is_last_level,
+              r.is_final_decision INTO app_id,
+              reviewer_id,
+              assignment_id,
+              stage_number,
+              level_num,
+              is_last_level,
+              is_final_decision
+          FROM
+              review r
+          WHERE
+              id = NEW.review_id;
+          templ_id = (
+              SELECT
+                  template_id
+              FROM
+                  application
+              WHERE
+                  id = app_id);
+          prev_status = (
+              SELECT
+                  status
+              FROM
+                  review_status_history
+              WHERE
+                  review_id = NEW.review_id
+                  AND is_current = TRUE);
+          INSERT INTO public.activity_log (type, value, application_id, "table", record_id, details)
+              VALUES ('REVIEW', NEW.status, app_id, TG_TABLE_NAME, NEW.id, json_build_object('prevStatus', prev_status, 'status', NEW.status, 'reviewId', NEW.review_id, 'reviewer', json_build_object('id', reviewer_id, 'name', (
+                              SELECT
+                                  full_name
+                              FROM "user"
+                              WHERE
+                                  id = reviewer_id), 'stage', json_build_object('number', stage_number, 'name', (
+                                      SELECT
+                                          title
+                                      FROM public.template_stage
+                                      WHERE
+                                          number = stage_number
+                                          AND template_id = templ_id))), 'sections', (
+                                  SELECT
+                                      json_agg(t)
+                                  FROM (
+                                      SELECT
+                                          title, code, "index"
+                                      FROM template_section
+                                  WHERE
+                                      code = ANY (ARRAY (
+                                              SELECT
+                                                  assigned_sections
+                                              FROM review_assignment
+                                          WHERE
+                                              id = assignment_id
+                                              AND template_id = templ_id
+                                              AND assigned_sections <> '{}'))
+                                      AND template_id = templ_id ORDER BY "index") t), 'level', level_num, 'isLastLevel', is_last_level, 'finalDecision', is_final_decision));
+          RETURN NEW;
+      END;
+      $application_event$
+      LANGUAGE plpgsql;
+        `)
```

Comment:
Did you change this intentionally? If not, can you revert it to how it was so it doesn't show up in the diff?  Thanks.

Reply:
You can also hide the "white space" changes using this:
![Screen Shot 2022-09-09 at 1 08 46 PM](https://user-images.githubusercontent.com/16461988/189251380-d72760d1-791c-4ade-a304-b383a7c32d33.png)